components/QuarantineProgress.js

```diff
@@ -32,7 +33,7 @@ export function QuarantineProgress({
 
   return (
     <View style={styles.container}>
-      <Text style={styles.title}>Quarantine Progress</Text>
+      <Text style={styles.title}>{i18n.t("labelQuarantineProgress")}</Text>
```

Comment:
You can create a component which gets id as a prop. 
`<Text id="labelQuarantineProgress" style={styles.title} />`